templates/register.html

```diff
@@ -0,0 +1,26 @@
+{% extends 'base.html' %}
+
+{% block title %}Регистрация - Мой Интернет-магазин{% endblock %}
+
+{% block content %}
+    <h2>Регистрация</h2>
+    <form action="{{ url_for('register') }}" method="post">
+        {{ form.hidden_tag() }}
+        <label for="first_name">Имя:</label>
+        {{ form.first_name(class="form-control") }}
+        <br>
+        <label for="last_name">Фамилия:</label>
+        {{ form.last_name(class="form-control") }}
+        <br>
+        <label for="email">Email:</label>
+        {{ form.email(class="form-control") }}
+        <br>
+        <label for="password">Пароль:</label>
+        {{ form.password(class="form-control") }}
+        <br>
+        <label for="confirm_password">Подтвердить пароль:</label>
+        {{ form.confirm_password(class="form-control") }}
+        <br>
+        <input type="submit" value="Зарегистрироваться" class="btn btn-primary">
```

Comment:
Что если у вас форма будет в несколько десятков полей. Не будете же вы для каждого поля писать по 3 строки
```suggestion
        {{ form.csrf_token }}
        {% for field in form if field.name != 'csrf_token' %}
        {{ field.label }}
        {{ field(class="form-control") }}
        <br>
        <ul class="alert alert-danger">
            {% for error in field.errors %}
            <li>{{ error }}</li>
            {% endfor %}
        </ul>
        <br>
        {% endfor %}
```
Причем строки
```html
        {% if field.errors %}
        <ul class="alert alert-danger">
            {% for error in field.errors %}
            <li>{{ error }}</li>
            {% endfor %}
        </ul>
        {% endif %}
```
как раз и есть обработка ошибок валидации формы.